---
sidebar_position: 5
---

# Next Steps

At this point, you should have everything you need to **perform downstream analysis** using your favorite **software packages**.

If you are unsure where to start, please consider contacting one of our statisticians for **[data analysis support](https://support.gedac.org/support/tickets/new)**.

If you would like to perform the analysis yourself or learn more about it, we have compiled a list of helpful resources below.

Anything **unclear** or **buggy** in this tutorial? [Please report it!](https://github.com/CSI-Genomics-and-Data-Analytics-Core/portal-docs/issues)

## Resources

- Analyzing RNA-seq data with [DESeq2](https://bioconductor.org/packages/3.21/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) (**highly recommended**)
- Differential analysis of sequence read count data using [edgeR](https://www.bioconductor.org/packages/devel/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf)
- Linear Models for Microarray and RNA-Seq Data using [LIMMA](https://www.bioconductor.org/packages/devel/bioc/vignettes/limma/inst/doc/usersguide.pdf)
- Detection of m6A from direct RNA-Seq data using [m6Anet](https://github.com/GoekeLab/m6anet)
- Identification and quantification of differential RNA modifications using [xPore](https://github.com/GoekeLab/xpore)
- Teaching materials at the [Harvard Chan Bioinformatics Core](https://github.com/hbctraining)
